recipe/meta.yaml

```diff
@@ -37,17 +39,19 @@ test:
 about:
   home: https://www.freetype.org/
   license: GPL-2.0-only and LicenseRef-FreeType
+  license_family: GPL
   license_file:
     - docs/GPLv2.TXT
     - docs/FTL.TXT
+  license_url: https://freetype.sourceforge.net/license.html
   summary: A Free, High-Quality, and Portable Font Engine
   description: |
     FreeType is designed to be small, efficient, highly customizable
     and portable while capable of producing high-quality output (glyph images)
     of most vector and bitmap font formats.
-  doc_url: http://freetype.sourceforge.net/freetype2/documentation.html
+  doc_url: https://freetype.sourceforge.net/freetype2/documentation.html
   doc_source_url: https://sourceforge.net/projects/freetype/?source=navbar
-  dev_url: https://gitlab.freedesktop.org/freetype/freetype
+  dev_url: https://git.savannah.gnu.org/cgit/freetype/
```

Comment:
Except for the source URL above, we should replace URL references to sourceforge.net and savannah.gnu.org with the [freedesktop.org GitLab instance](https://gitlab.freedesktop.org/freetype/freetype), which according to the [FreeType developer's page](https://freetype.org/developer.html) is the canonical repository for this project.